docs/contents/releases/1-22/12/CHANGELOG-v1-22-eks-12.md

```diff
@@ -0,0 +1,20 @@
+# Changelog for v1-22-eks-12
+
+This changelog highlights the changes for [v1-22-eks-12](https://github.com/aws/eks-distro/tree/v1-22-eks-12).
+
+## Changes
+
+### Patches
+* Updated to 1.22.15 and added patch ([1327](https://github.com/aws/eks-distro/pull/1327))
+
+### Projects
+* Updated to 1.22.15 and added patch ([1327](https://github.com/aws/eks-distro/pull/1327))
+* Update cni to v0.9.0 for 1.21 and 1.22 ([1301](https://github.com/aws/eks-distro/pull/1301))
+* Updated kubernetes-csi/external-attacher to v4.0.0 ([1275](https://github.com/aws/eks-distro/pull/1275))
+* Updated kubernetes-csi/external-provisioner to v3.3.0 ([1274](https://github.com/aws/eks-distro/pull/1274))
+* Updated kubernetes-csi/external-resizer to v1.6.0 ([1270](https://github.com/aws/eks-distro/pull/1270))
+* Updated kubernetes-csi/external-snapshotter to v6.1.0 ([1269](https://github.com/aws/eks-distro/pull/1269))
+
+### Base Image
+No changes since last release
```

Comment:
```suggestion
### Patches
No changes since last release

### Projects
No changes since last release

### Base Image
Updated base image ([#1343](https://github.com/aws/eks-distro/pull/1343) and [#358](https://github.com/aws/eks-distro/pull/1358))
```